lms/djangoapps/grades/api/v0/tests/test_views.py

```diff
@@ -0,0 +1,203 @@
+"""
+Tests for the views
+"""
+from datetime import datetime
+
+from django.core.urlresolvers import reverse
+from mock import patch
+from pytz import UTC
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from student.tests.factories import UserFactory
+from student.models import CourseEnrollment
+from xmodule.modulestore.tests.factories import CourseFactory, ItemFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase, TEST_DATA_SPLIT_MODULESTORE
+
+
+class CurrentGradeViewTest(SharedModuleStoreTestCase, APITestCase):
+    """
+    Tests for the Current Grade View
+
+    The following tests assume that the grading policy is the edX default one:
+    {
+        "GRADER": [
```

Comment:
Thanks for the explanation.  I sometimes have higher expectations of our platform and mistakenly assumed that this was documented somewhere.  :)

If you want, you can also override and simplify the grading policy used by your tests.  For example, see the tests in https://github.com/edx/edx-platform/blob/master/lms/djangoapps/courseware/tests/test_submitting_problems.py#L377